### DISCUSSION TITLE
v0.4.3

### DISCUSSION THREAD

**Discussion by littlesera** [littlesera/LSCG#509] [Announcements]
## What's Changed
* Refactor Leashing by @littlesera in https://github.com/littlesera/LSCG/pull/489
* Hypnotic Suggestions by @littlesera in https://github.com/littlesera/LSCG/pull/491
* Hypno Suggestion polish + Updates from Dev by @littlesera in https://github.com/littlesera/LSCG/pull/508

**Full Changelog**: https://github.com/littlesera/LSCG/compare/v0.3.54...v0.4.3

<hr /><em>This discussion was created from the release <a href='https://github.com/littlesera/LSCG/releases/tag/v0.4.3'>v0.4.3</a>.</em>

**Comment by Epetuk666** [comment]
Hypnotic Suggestions
Either it seems to me or there is a mistake here.
"blocked instruction" Has the opposite effect. Until I choose a blocked instruction for myself, it constantly writes to me that I am "You are immune to the suggestion's "Orgasm" instruction and shake some of player influence". If I select this instruction as blocked, everything starts to work. Sorry if I am writing in the wrong place.

**Reply by littlesera** [reply]
Resolved in #512
